labman/gui/templates/normalization.html

```diff
@@ -0,0 +1,183 @@
+{% extends sitebase.html %}
+
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery-ui.min.css" type="text/css"/>
+<script src="/static/vendor/js/jquery-ui.min.js" type="text/javascript"></script>
+
+<script src="/static/js/reagentModal.js"></script>
+
+<script type='text/javascript'>
+
+  function normalize_send() {
+    // Get the plate ids
+    var plateIds = []
+    for (var item of $('#plate-list').children()) {
+      plateIds.push(item.getAttribute('pm-data-plate-id'));
+    }
+
+    $.post('/process/normalize', {
+        'plate_id': plateIds[0],
+        'water': $('#water-input').val(),
+        'plate_name': $('#plate-name').val(),
+        'total_vol': $('#total-volume').val(),
+        'ng': $('#ng-input').val(),
+        'min_vol': $('#min-input').val(),
+        'max_vol': $('#max-input').val(),
+        'resolution': $('#resolution-input').val(),
+        'reformat': $('#reformat-input').val()
+      }, function(data) {
+      bootstrapAlert('Information saved', 'success');
```

Comment:
After discussion with the group, we think that after a successful post, we need to "disable" all the inputs from the HTML, and show a button to "Download pick-list". The class NormalizationProcess has a function generate_echo_picklist to generate that file on the fly. That would need to have it's own endpoint, which can be "/process/normalize/([0-9]+)/echo_pick_list$" that returns the file. An example of such handler can be seen here: https://github.com/jdereus/labman/blob/master/labman/gui/handlers/process_handlers/sequencing_process.py#L46

Also, it would be ideal if these tasks can be encapsulated in a function, since we will be able to re-use it later.

Reply:
k, working on this.